src/hotspot/share/ci/bcEscapeAnalyzer.hpp

```diff
@@ -98,8 +98,8 @@ class BCEscapeAnalyzer : public ResourceObj {
   void initialize();
   void clear_escape_info();
   void compute_escape_info();
-  vmIntrinsics::ID known_intrinsic();
-  void compute_escape_for_intrinsic(vmIntrinsics::ID iid);
+  vmIntrinsicID known_intrinsic();
+  void compute_escape_for_intrinsic(vmIntrinsicID iid);
```

Comment:
I think the use of vmIntrinsics::ID in bcEscapeAnalyzer.cpp should also be changed to vmIntrinsicID for consistency, even though they are the same type. (The same for other hpp files such as ciMethod.hpp)